### DISCUSSION TITLE
Discuss computer vision solutions

### DISCUSSION THREAD

**Discussion by Alintya** [CoRoSoN-Kit/CoRoSoN-Kit#49] [Ideas]
In continuation of #34

**Comment by Vision-Kek** [comment]
Consider recent posts in https://junior.forum.robocup.org/t/feedback-on-possible-changes-to-camera-rules.
What kind (single/multicam/360deg/lens/mirror) of vision systems we choose to support?

I suppose we expect teams to make optical apparatus themselves? Then I don't know trivial solutions for beginners, so there would be only the straight-cam approach for them. Then pixy with its built-in functions is probably enough and easiest. Either way, processing must happen on camera device, we can't load full frames in pico memory. With OpenMV we'd be more flexible since we can implement custom algorithms and api. I think it is no large problem to switch later, maybe we keep it simple for v1
